paperqa/docs.py

```diff
@@ -60,7 +60,7 @@ async def empty_callback(result: LLMResult) -> None:
     pass
 
 
-async def print_callback(result: LLMResult) -> None:
+async def print_callback(result: str | LLMResult) -> None:
```

Comment:
Arguably we should just remove this `print_callback` altogether, not sure why we have it